src/lib.rs

```diff
@@ -142,38 +142,29 @@ pub fn version() -> u32 { unsafe { bitcoinconsensus_version() as u32 } }
 ///
 /// # Arguments
 ///
-///  * spend_output_script: A Bitcoin transaction output script to be spent, serialized in Bitcoin's on wire format.
+///  * spend_output: A Bitcoin transaction output script to be spent, serialized in Bitcoin's on wire format.
 ///  * amount: The spent output amount in satoshis.
 ///  * spending_transaction: The spending Bitcoin transaction, serialized in Bitcoin's on wire format.
 ///  * input_index: The index of the input within spending_transaction.
 ///
 /// # Examples
 ///
-/// The (randomly choosen) Bitcoin transaction [aca326a724eda9a461c10a876534ecd5ae7b27f10f26c3862fb996f80ea2d45d](https://blockchain.info/tx/aca326a724eda9a461c10a876534ecd5ae7b27f10f26c3862fb996f80ea2d45d)
-/// spends one input, that is the first output of
-/// [95da344585fcf2e5f7d6cbf2c3df2dcce84f9196f7a7bb901a43275cd6eb7c3f](https://blockchain.info/tx/95da344585fcf2e5f7d6cbf2c3df2dcce84f9196f7a7bb901a43275cd6eb7c3f)
-/// with a value of 630482530 satoshis.
-///
-/// The spending transaction in wire format is:
-///
+/// The (randomly choosen) Bitcoin transaction [aca326a7] spends one input, that is the first output
+/// of [95da3445]. The spending transaction in wire format is:
```

Comment:
In 28622081f8d8f17bae8b68db353d4f772193ac08:

I don't really like the idea of using short hashes, since you can't look them up anywhere.

But if we are going to touch this code, we should drop the links to bc.i. They are a hostile actor. If we must link to a block explorer we can use blockstream.info. Or nothing.

Reply:
Fair, and fair. Will re-spin. Thanks

Reply:
Remove links and used the whole hashes.